### PR TITLE
All Sonar Lint errors cleared

### PR DESCRIPTION
**Pull request by mich2635**

### CODE REVIEW

File: src/main/java/in/micheal/dao/DebtUserDetailsDAO.java [hunk 1]
```diff
@@ -229,8 +234,8 @@ public static void deleteAllRecords() throws DbException {
 
 			rs = pst.executeQuery();
 			while (rs.next()) {
-				String bookname = rs.getString("taken_book");
-				int bookquantity = rs.getInt("taken_quantity");
+				String bookname = rs.getString(BOOK);
+				int bookquantity = rs.getInt(QUANTITY);
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/micheal/dao/DebtUserDetailsDAO.java]: follow naming convention

File: src/main/java/in/micheal/dao/DebtUserDetailsDAO.java [hunk 2]
```diff
@@ -74,7 +79,7 @@ public static Long findDebtUser(Long userID, String bookName) throws DbException
 
 			rs = pst.executeQuery();
 			while (rs.next()) {
-				userid = rs.getLong("user_id");
+				userid = rs.getLong(USER_ID);
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/micheal/dao/DebtUserDetailsDAO.java]: follow naming convention

File: src/main/java/in/micheal/dao/DebtUserDetailsDAO.java [hunk 3]
```diff
@@ -18,6 +18,11 @@ private DebtUserDetailsDAO() {
 		// default constructor
 	}
 
+	static final String BOOK = "taken_book";
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/micheal/dao/DebtUserDetailsDAO.java]: Add private to reduce the scope of the variable.

File: src/main/java/in/micheal/dao/DebtUserDetailsDAO.java [hunk 4]
```diff
@@ -279,7 +284,7 @@ public static void deleteAllRecords() throws DbException {
 			}
 
 		} catch (ClassNotFoundException | SQLException e) {
-			throw new DbException("UNABLE TO SEARCH BOOKS");
+			throw new DbException("UNABLE TO GET RECORDS ,TRY AGAIN");
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/micheal/dao/DebtUserDetailsDAO.java]: Error messages manage in a separate MessageConstant file.

**Review by ghost — CHANGES_REQUESTED** [review]
Added review comments.

File: src/main/java/in/micheal/dao/DebtUserDetailsDAO.java (hunk 2 above)
Comment (mich2635) [reply src/main/java/in/micheal/dao/DebtUserDetailsDAO.java]: ok sir,will do

File: src/main/java/in/micheal/dao/DebtUserDetailsDAO.java (hunk 3 above)
Comment (mich2635) [reply src/main/java/in/micheal/dao/DebtUserDetailsDAO.java]: ok sir

File: src/main/java/in/micheal/dao/DebtUserDetailsDAO.java (hunk 1 above)
Comment (mich2635) [reply src/main/java/in/micheal/dao/DebtUserDetailsDAO.java]: ok sir

**Review by ghost — APPROVED** [review]

**Pull request merged**